travharv/executor.py

```diff
@@ -32,6 +33,9 @@ def __init__(
         self.NSM = NSM
         self.tasks = tasks
         self.rdf_store_access = rdf_store_access
+        self.execution_report = ExecutionReport(
+            rdf_store_access, config_filename
```

Comment:
why is the store needed in the Report?

why not let the Report just accumulate info, then read / convert it to turtle / parse into rdflib graph and add to store?